processing/src/main/java/io/druid/query/groupby/epinephelinae/Grouper.java

```diff
@@ -159,6 +160,14 @@ public String toString()
      * Create a new KeySerde, which may be stateful.
      */
     KeySerde<T> factorize();
+
+    /**
+     * Return an object that knows how to compare two serialized key instances. Will be called by the
+     * {@link #iterator(boolean)} method if sorting is enabled.
+     *
+     * @return comparator for keys
```

Comment:
this is actually the comparator for deserialized key objects, updated the comment to say, "key objects"